stream-chat-android-offline/src/main/java/io/getstream/chat/android/offline/experimental/plugin/factory/StreamOfflinePluginFactory.kt

```diff
@@ -88,6 +91,19 @@ public class StreamOfflinePluginFactory(
         val userStateFlow = MutableStateFlow(ChatClient.instance().getCurrentUser())
         val stateRegistry = StateRegistry.getOrCreate(scope, userStateFlow, repos, repos.observeLatestUsers())
         val logic = LogicRegistry.getOrCreate(stateRegistry)
+        val globalState = GlobalMutableState.getOrCreate()
+
+        val defaultInterceptor = DefaultInterceptor(
+            sendMessageInterceptor = SendMessageInterceptorImpl(
+                context = appContext,
+                logic = logic,
+                globalState = globalState,
+                scope = chatDomainImpl.scope,
```

Comment:
`scope` and `repos` are already present in this method there's no need to access it from chatDomainImpl